docs/api_reference.md

```diff
@@ -217,6 +218,14 @@ These are all available on the global `LookerCharts.Utils` object.
 
 	This function accepts a cell and returns a Looker advanced filter syntax string that would match the value of this cell.
 
+- `LookerCharts.Utils.crossfilter({row, pivot, event})`
```

Comment:
Armando changed this recently to `toggleCrossfilter` so  we now have: `LookerCharts.Utils.toggleCrossfilter`

Here's a simple example we could add as well:
```
... add event listener to element and check if crossfiltering is enabled when event occurs ... 

.on("click", function (row, event) {
  if (details.crossfilterEnabled) {
    LookerCharts.Utils.toggleCrossfilter({
      row: row,
      event: event,
    });
  } else { 

...  or do something else (like open the drill menu) ...
```